p/poetry/build_info.json

```diff
@@ -0,0 +1,16 @@
+{
+  "maintainer": "aastha-sharma2",
+  "package_name": "poetry",
+  "github_url": "https://github.com/python-poetry/poetry.git",
+  "version": "1.8.3",
+  "wheel_build" : true,
+  "default_branch": "main",
+  "package_dir": "p/poetry",
+  "build_script": "poetry_ubi_9.3.sh",
+  "validate_build_script": "true",
+  "use_non_root_user": "false",
+  "docker_build": false,
+  "*.*.*": {
```

Comment:
Can't we simply set it "*", why to complicate?

Reply:
> Can't we simply set it "*", why to complicate?

Off Course we can, changing it.